entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+trap stop_gracefully TERM INT
+
+stop_gracefully(){
+  echo "stopping ampd process"
+  killall "ampd"
+  sleep 5
+  echo "ampd process stopped"
+}
+
+/usr/local/bin/ampd &
+wait
+exec "$@"
```

Comment:
What are some examples of commands that could be executed in this context?

Reply:
A common use case is `docker run -it ampd /bin/bash`
If ampd process exit with error, it will exec into container immediately, make it easier to debug within container environment.